client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-button.jsx

```diff
@@ -33,20 +33,20 @@ export default React.createClass( {
 
 	render() {
 		const { site, redirect, linkDisplay } = this.props;
-		const buttonElement = linkDisplay ? 'button' : Button;
 
 		const buttonClasses = classNames( {
-			button: true,
-			'disconnect-jetpack-button': true,
-			'is-link': linkDisplay
+			'disconnect-jetpack-button': true
 		} );
 
-		const buttonProps = assign( {}, this.props, {
+		const omitProps = [ 'site', 'redirect', 'isMock', 'linkDisplay', 'text' ];
+		const buttonProps = {
+			...omit( this.props, omitProps ),
 			id: `disconnect-jetpack-${ site.ID }`,
 			className: buttonClasses,
```

Comment:
At this point, you could simply inline the `disconnect-jetpack-button` class and drop the `classNames` dependency.